docs/doc_5_best_practices.md

```diff
@@ -28,6 +28,45 @@ if test != ""
     requestBody.StringPointerValue = test
 ```
 
+## Working with Arrays
```

Comment:
the examples are really with Slices, or we can say Working with Arrays and Slices